plugins/kotlin/jvm-debugger/core/kotlin.jvm-debugger.core.iml

```diff
@@ -53,5 +53,7 @@
     <orderEntry type="module" module-name="intellij.java.debugger.impl" />
     <orderEntry type="module" module-name="intellij.java.psi.impl" />
     <orderEntry type="module" module-name="kotlin.code-insight.utils" />
+    <orderEntry type="library" name="google-dexlib2" level="project" />
```

Comment:
This library is currently used only in Android plugin. What do you think of adding an extension point with implementation in android plugin?